models/UserModel.js

```diff
@@ -30,10 +15,11 @@ const hashPassword = (plainPassword) =>
 
 // Function to validate the unicity of a user username
 const usernameAlreadyExists = async (username) =>
-  !!(await User.findOne({ username }));
+  !!(await prisma.user.findFirst({ where: { username } }));
```

Comment:
[Prisma's doc](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findunique)
```suggestion
  !!(await prisma.user.findUnique({ where: { username } }));
```